tests/std/tests/P0088R3_variant/test.cpp

```diff
@@ -6971,6 +6971,27 @@ namespace msvc {
                 using R = immobile_data;
                 assert(std::visit<R>(std::identity{}, std::variant<int, short>{13}).x == 13);
                 assert(std::visit<R>(std::identity{}, std::variant<int, short>{short{42}}).x == 42);
+
+                // Verify that conversions to an object is not copied/moved are correctly handeled
+                struct convertible_to_immobile_one {
+                    operator immobile_data() const {
+                        return immobile_data{1729};
+                    }
+                };
+
+                struct convertible_to_immobile_other {
+                    operator immobile_data() const {
+                        return immobile_data{1138};
+                    }
+                };
+
+                using VarTestConv = std::variant<convertible_to_immobile_one, convertible_to_immobile_other>;
+                // TRANSITION, DevCom-10024578

```

Comment:
This was marked as fixed in 17.3 Preview 3, but it appears that the workaround is still necessary. We should report a new bug.

Reply:
I've filed DevCom-10111923 which is identical to DevCom-10024578 except for using `__is_convertible_to(B, A)` instead of `__is_constructible(A, B)`. I'll update the PR.

Reply:
This should be now DevCom-10112408.

@CaseyCarter has reported DevCom-10111923, but IIUC it doesn't block the original test case. Since `_Invoke_convertible` is using handmade SFINAE instead of that intrinsic.